.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
```

Comment:
I think we can skip `main` for now. On `master` we will have a different pipeline. This workflow should be just for `develop`.